lib/DropdownMenu/styles.scss

```diff
@@ -71,9 +72,9 @@ $dropdown-menu-link-padding: 7px 15px !default;
   padding: $dropdown-menu-link-padding;
   text-decoration: none;
   cursor: pointer;
-  color: #6f7c82;
-  font-weight: $typo-weight-roman;
-  font-size: $typo-size-small;
+  color: $color-text-base;
+  font-weight: $typo-weight-semi-bold;
+  font-size: $dropdown-menu-font-size;
```

Comment:
Have updated the styling in this file to reflect the update to all dropdowns menus